src/libaktualizr/uptane/CMakeLists.txt

```diff
@@ -64,8 +64,7 @@ add_aktualizr_test(NAME uptane SOURCES uptane_test.cc PROJECT_WORKING_DIRECTORY)
 set_tests_properties(test_uptane PROPERTIES LABELS "crypto")
 
 add_aktualizr_test(NAME uptane_delegation SOURCES uptane_delegation_test.cc PROJECT_WORKING_DIRECTORY
-                   ARGS ${PROJECT_BINARY_DIR}/uptane_repos)
-add_dependencies(t_uptane_delegation uptane_repo_delegation_basic)
```

Comment:
Too bad that we can't generate these through CMake anymore, as it allows an easy inspection. Could we still do it for the basic test without revocation?

In any case, there should be a line `add_dependencies(t_uptane_delegation aktualizr-repo)` if we call the executable directly.

Reply:
With `add_dependencies(t_uptane_delegation aktualizr-repo)` agree. But about `delegation_basic.sh`, I think it is not a problem to run it manually when needed. Otherwise we will have different ways to generate delegations. What do you think?

Reply:
Ok let's keep that for now, thanks for fixing the dependency